docusaurus/docs/reference/tunnelers/linux/index.md

```diff
@@ -20,15 +21,18 @@ Reasons to use the package:
 1. Automatically enroll the identity and clean up the enrollment token in identity directory.
 1. Automatically upgrade the tunneller when a new package is available.
 
-Linux DEB packages are currently available for the x86_64 and arm64 platforms and RPM packages are available for x86_64. Additionally, there are executable downloads available for arm/v7 (32bit) for [manual installation](#manual-installation).
+Linux DEB packages are currently available for the x86_64 and arm64 platforms and RPM packages are available for x86_64.
+Additionally, there are executable downloads available for arm/v7 (32bit) for [manual
+installation](#manual-installation).
 
 :::note
-It is not necessary to manually enroll the identity when using the RPM or DEB package. Just install the token in the identities directory with file owner "ziti" and it will be enrolled and cleaned up when you start the service.
+It is not necessary to manually enroll the identity when using the RPM or DEB package. Install the token in the
```

Comment:
this note is out of place. The user hasn't even installed the binary yet and you're talking about identities. This should be down in the "Adding an Identity" section